plugins/osdn/osdn.go

```diff
@@ -114,8 +114,8 @@ func (oi *OsdnRegistryInterface) GetNodes() ([]osdnapi.Node, error) {
 		return nil, err
 	}
 
-	nodes := make([]osdnapi.Node, 0)
-	for _, node := range knodes.Items {
+	nodes := make([]osdnapi.Node, len(knodes.Items))
```

Comment:
can do `nodes := make([]osdnapi.Node, 0, len(knodes.Items))`, to preallocate the desired capacity, but set length to 0, then just use `append` normally below without bothering with the indexes. That makes it safe if there are code changes in the future that make us exceed the expected capacity, or conditionally skip one of the items and not end up using all the expected capacity